### DISCUSSION TITLE
Extending Disk for Dokku's Postgres with Digital Ocean Block Storage

### DISCUSSION THREAD

**Discussion by ohld** [dokku/dokku#4692] [Q&A]
I'm using Dokku's Postgres plugin on DigitalOcean droplet with 160GB disk. I want to expand the disk using DO's block storage but I'm not sure if that is even possible. I want to leave my current Postgres data as is and just attach more volume.

Is it possible to attach DigitalOcean's block storage to already created Dokku's Postgres? So just to enlarge the space for Postgres needs.

It seems like this is not Dokku-related question but more Docker-Postgres-related question.

**Comment by josegonzalez** [comment]
No it isn't possible. You'll need to move your data to that attached storage and then point the service's data dir to that drive (either via symlink or fstab mount).

**Comment by muZk** [comment]
I just did it, and I think it was successful 😅

In case it helps anyone, there are the steps:

Note: Postgres service name is `db` and my volume name is `volume_name`.

1. I created a volume for the droplet where dokku was installed and has Postgres running. I did it via the web interface by clicking "Add Volume".
2. I started the droplet again and entered via ssh. I realized the volume was already mounted by running `df -h`. Take note of the path, it should be something like `/mnt/volume_name`
3. I stopped the Postgres service: `dokku postgres:stop db`
4. I used the following command to know where the data folder stored for my Postgres `service: dokku postgres:info db --data-dir` (take note of this).
5. I created a folder to store this data in my volume: `cd /mnt/volume_name && mkdir data`
6. `mv /var/lib/dokku/services/postgres/db/data /mnt/volume_name/db` to move Postgres data to the volume
7. `sudo ln -s /mnt/volume_name/data /var/lib/dokku/services/postgres/db/data` to create the symlink
8. `dokku postgres:start db` to start the service
